public/index.html

```diff
@@ -15,6 +15,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" href='../src/index.css'>
```

Comment:
why do we need it?

Reply:
I fixed it, there is no reason to use, I use Global Styles from styled-components